sapi/apache2handler/config.m4

```diff
@@ -58,9 +63,10 @@ if test "$PHP_APXS2" != "no"; then
 
   dnl Test that we're trying to configure with apache 2.x
   PHP_AP_EXTRACT_VERSION($APXS_HTTPD)
-  if test "$APACHE_VERSION" -lt 2000044; then
+  AS_VERSION_COMPARE([$APACHE_VERSION], [2000044], [
```

Comment:
Yes, this should be updated also at some point. Also for Windows. To something like 2.4.

Reply:
I'll open a separate discussion for dropping support for EOL Apache branches 2.0 and 2.2. I'm not fully sure yet what all this can impact, but Apache 2.2 is definitely EOL since 2018 and shouldn't be encouraged to use anyway.

Reply:
Apache minimum version bump: https://github.com/php/php-src/pull/14664
I'm still checking all of this...